README.md

```diff
@@ -155,10 +155,23 @@ PS > $ram = $client.remote_accesses() | Select-Object -Last 1
 PS > $client.delete_remote_access($ram.id)
 ```
 
+- Use the client to retry updates on a specific server (here 2 updates identified by its ID):
```

Comment:
```
Use the client to scedule updates on a specific server (here 2 updates identified by its IDs). 
```
add optional parameters in example, and mention that without the the server's deployment policy is used.